pkg/key/key.go

```diff
@@ -23,10 +23,22 @@ func APIEndpoint(id string, base string) string {
 	return fmt.Sprintf("api.%s.k8s.%s", id, base)
 }
 
+func DomainFromHost(h string) string {
```

Comment:
The key functions are used with a lot of assumptions specific to `awscnfm` and how we may execute it. We need to see if this works out well, so we need to make some experiences with this approach over time.